lib/Runtime/ByteCode/OpLayoutsAsmJs.h

```diff
@@ -61,6 +61,9 @@ namespace Js
         return (uint)(Js::OpCodeAsmJs::Simd128_End - Js::OpCodeAsmJs::Simd128_Start) + 1 + (uint)(Js::OpCodeAsmJs::Simd128_End_Extend - Js::OpCodeAsmJs::Simd128_Start_Extend) + 1;
     }
 
+    inline bool IsByteOpCode(Js::OpCodeAsmJs op)        { return op < ((uint)Js::OpCodeAsmJs::MaxByteSizedOpcodes + 1); }                                       /* op == [0 - 256) ?       */
+    inline bool IsExtendedOpCode(Js::OpCodeAsmJs op)    { return !IsByteOpCode(op) && (uint)op < 2 * ((uint)Js::OpCodeAsmJs::MaxByteSizedOpcodes + 1); }        /* op == [0 - 2 * 256) ?  */
```

Comment:
Nit: misleading comment. Should read: op == [256 - 2 * 256) ? Same for the one below.